### DISCUSSION TITLE
Domoticz option not available on Tasmota-lite

### DISCUSSION THREAD

**Discussion by jepe63** [arendst/Tasmota#14677] [Q&A] [answered]
Hi , i flashed my sonoffs basic from 6.4 -7 - 8 - 9- to 10 as indicated on the website, al working fine except the option of domoticz is not there anymore, did i do something wrong ?
![tas1](https://user-images.githubusercontent.com/19386647/151699838-79ec4a49-13f0-4bfb-ae8f-652b48eca731.png)
![tas2](https://user-images.githubusercontent.com/19386647/151699840-13a523b4-2482-4b68-acf3-50836569d781.png)
![tas3](https://user-images.githubusercontent.com/19386647/151699842-673d23ce-a0e2-4fa8-9cc3-90c885dd3243.png)
![tas4](https://user-images.githubusercontent.com/19386647/151699843-03fdce83-44e4-4f7e-8207-6ef42386ba70.png)

**Comment by barbudor** [comment, marked as answer]
What is the last binary you flashed?
If it tasmota-lite you need to upgrade to the full tasmota

**Reply by jepe63** [reply]
Yeah i know unfortunally nog possible, gives error too large

**Reply by ascillato** [reply]
Upgrade to Tasmota-minimal and after that, upgrade to tasmota.bin

Or, easier. Go to upgrade firmware and click on upgrade firmware on the OTAurl frame. That will automatically download and install minimal and then, it will download and install tasmota.bin

**Reply by sfromis** [reply]
And when upgrading, make sure to select the compressed .gz version (as long as you are not still on 6.4)
